css/opengazettes.scss

```diff
@@ -0,0 +1,3 @@
+.jumbotron h1 {
+  color: fuchsia !important;
```

Comment:
Cool. There are a bunch of variables so before I send @pudo a pull request I'll let @smmbll loose on it so he can define better what's needed. This is an unrealistic example.